x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/dataframe/DataFrameAnalyticsConfigTests.java

```diff
@@ -88,6 +93,88 @@ protected DataFrameAnalyticsConfig createTestInstance() {
         return createRandom(randomValidId(), lenient);
     }
 
+    @Override
+    protected List<Version> bwcVersions() {
+        return AbstractBWCSerializationTestCase.getAllBWCVersions(Version.V_7_7_0);
```

Comment:
why is it overridden? why is `DEFAULT_BWC_VERSIONS` not sufficient?
(if possible add the answer as code comment)

Reply:
It is overridden so these tests would actually do something :). I will remove the override on backport (so the BWC tests run according to the 7.series rules) and then the override can be removed in 8.series.